src/main.js

```diff
@@ -227,7 +227,12 @@ module.exports = {
   async fixJob(isSave = false) {
     const textEditor = atom.workspace.getActiveTextEditor()
 
-    if (!textEditor || textEditor.isModified()) {
+    if (!textEditor || !atom.workspace.isTextEditor(textEditor)) {
+      // Silently return if the TextEditor is invalid
+      return
```

Comment:
We talked about maybe logging something out to the console, did you decide not to?